composer.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "mantisbt/mantisbt",
+    "description": "Mantis Bug Tracker",
+    "type": "project",
+    "require": {
+        "slim/slim": "^3.0"
+    },
+    "license": "GPL v2",
+    "authors": [
+        {
+            "name": "Victor Boctor",
+            "email": "vboctor@gmail.com"
+        }
+    ]
```

Comment:
We can add more information as we consumer.  Composer has a bunch of schema to accommodate for different usages.  For example, customizing auto-load map, fetching published packages or packages from repositories, etc.  I don't want to pack stuff that is not needed for this specific task.  I've added a link to our website in the author information though.